### DISCUSSION TITLE
Rename layers

### DISCUSSION THREAD

**Discussion by heatgithub** [DomCR/ACadSharp#499] [Q&A] [answered]
I haven't found a way to rename a layer, if there is a way I would be glad if someone can show me how to accomplish that. Otherwise that would be a nice feature in the near future.

**Comment by DomCR** [comment, marked as answer]
Hi @heatgithub,

To change a Layer name, you just have to change the property name:

```C#
Layer.Name = "new_name";
```

Here is a test as an example:
https://github.com/DomCR/ACadSharp/blob/4856decca5e1b3d79f85a0cf9020e951d0d6aa7c/src/ACadSharp.Tests/Tables/TableEntryTests.cs#L31

Hope this helps.

**Reply by heatgithub** [reply]
Oh, too easy! :)
I was looking for Rename or something.
